File: Java/_06SwapNumbers.java
package Java;

import java.util.Scanner;

public class _06SwapNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int num1 = scanner.nextInt();
        int num2 = scanner.nextInt();
        scanner.close();

        System.out.println("num1: " + num1 + " num2: " + num2);

        int temp = num1;
        num1 = num2;
        num2 = temp;

        System.out.println("num1: " + num1 + " num2: " + num2);
    }
}

File: Kotlin/_06SwapNumbers.kt
fun main() {
    var num1 = readln().toInt()
    var num2 = readln().toInt()

    println("$num1\n$num2")

    val temp = num1
    num1 = num2
    num2 = temp

    println("$num1\n$num2")
}